app/views/users/registrations/_single_school.html.erb

```diff
@@ -25,5 +25,5 @@
   </tbody>
 </table>
 
-<%= govuk_button_link_to "Confirm", users_register_path(email: @email, school_id: @school) %>
-<%= govuk_button_link_to "No, change details", users_check_email_path, class: "govuk-button--secondary" %>
+<%= govuk_link_to "Confirm", users_register_path(email: @email, school_id: @school), button: true %>
```

Comment:
Just curious why this has been changed

Reply:
We added the `govuk-components` gem and stripped out our own `govuk_button_link_to` helper method